core/storage/storage.go

```diff
@@ -17,9 +17,13 @@
 
 package storage
 
+import "github.com/asdine/storm"
+
 // Storage stores persistent objects for future usage
 type Storage interface {
 	Store(issuer string, data interface{}) error
+	GetAll(issuer string, data interface{}) error
 	Delete(issuer string, data interface{}) error
 	Close() error
+	GetDB() *storm.DB
```

Comment:
this makes the whole interface rather useless. The storage is now coupled to storm DB. If we change the storage, we'll still have to implement a getdb method that returns a storm db. If you absolutely must have the "GetDB" method, you should then have the method return an interface{} and cast it to the specific DB driver on the consumer side(for instance, tests). In this case I don't think we even need the GetDB

Reply:
removed GetDB, added needed storage functions to storage interface